addon/components/rdfa-editor-container.hbs

```diff
@@ -0,0 +1,45 @@
+{{#if @busy}}
+  <AuBodyContainer>
+    <div class="au-c-rdfa-editor">
+      <div class="say-container say-container--sidebar-left say-container--paper say-container--sidebar-right">
+        <div class="say-container__main">
+          <div class="say-editor rdfa-annotations rdfa-annotations-highlight rdfa-annotations-hover">
+            <div class="say-editor__paper">
+              <div class="au-c-scanner">
+                <div class="au-c-scanner__text">
+                  <AuLoader @padding="small"/>
+                  <AuHelpText @size="large">{{if @busyText @busyText
+                                                 (t "rdfaEditorContainer.defaultBusyText") }}
+                  </AuHelpText>
+                </div>
+                <span class="au-c-scanner__bar"></span>
+              </div>
+              <div class="say-editor__inner say-content">
+                {{html-safe this.editor.htmlContent}}
```

Comment:
This helper doesn't exist in the addon so it crashes if you set `@busy` to `true`.